.github/CODEOWNERS

```diff
@@ -195,6 +205,7 @@
 
 # Scene
 
+/scene/                                       @godotengine/_systems
```

Comment:
Should probably also be `core`

Reply:
I don't think that's necessary for this root container, it's pretty limited in scope

Reply:
Arguably `scene/property_utils.h` and `scene/property_list_helper.h` is pretty good to keep track of

Reply:
Ohhh, that's a fair point